store/data.js

```diff
@@ -210,7 +215,18 @@ export const actions = {
   ) {
     try {
       commit('setTransactionsLoading', true)
-      const transactions = await api.getTransactions(address, pageNumber)
+      const transactionsOrigin = await api.getTransactions(address, pageNumber)
+      if (isValidLikeAddress(address)) {
+        address = changeAddressPrefix(address, 'cosmos')
+      } else if (isValidCosmosAddress(address)) {
+        address = changeAddressPrefix(address, 'like')
```

Comment:
`address` is an input param so prefer not changing it. Please name a new variable `prefixChangedAddress` for use